packages/app/src/cli/models/extensions/extension-instance.test.ts

```diff
@@ -244,28 +243,6 @@ describe('bundleConfig', async () => {
       }),
     )
   })
-
```

Comment:
Removed this test because we in our `config` for `webhooks` and `webhook_subscription` specs, we don't return `subscriptions` any more, just the subscription object itself